### ISSUE TITLE
Require is not defined

### ISSUE THREAD

**Issue by jdschuitemaker** [EddyVerbruggen/SocialSharing-PhoneGap-Plugin#268]
When I debug my app that uses the SocialSharing plugin I see a message saying:
Uncaught ReferenceError: require is not defined 

Is this expected and if so, what action should I take? I Googled for require and it seems that this has to do with node.js.

Thank you in advance!

**Comment by KhaledElAnsari** [comment]
Did you guys found a solution for this problem? because I'm facing it now!

**Comment by jdschuitemaker** [comment]
If I recall well, my issue was that I referenced socialsharing.js in my
index.html file. That is not required for this plug-in. Could it be that
you are experiencing the same?

Met vriendelijke groeten, Warm regards,

Jan-Dirk Schuitemaker

**Comment by KhaledElAnsari** [comment]
My problem was with the Cordova file src , It was inside a folder called JS so the src was "src=JS/cordova.js" but I put it back in the main folder (src = "cordova.js") and it work so the other functions ... thanks for your help man :)

**Comment by somnaths** [comment]
Thanks @KhaledElAnsari . 
Your reply (moving cordova.js back to the root ) solved my issue also.

**Comment by amitpatil321** [comment]
@EddyVerbruggen Now i am facing the same issue, getting the same error. I checked solution provided by @KhaledElAnsari and my cordova.js file is in right place. I created fresh app and it worked well with it, Its just giving me "require" error with my existing app. I am listing list of plugins i am using
com.ionic.keyboard 1.0.4 "Keyboard"
cordova-plugin-console 1.0.1 "Console"
cordova-plugin-device 1.0.1 "Device"
cordova-plugin-splashscreen 2.1.0 "Splashscreen"
cordova-plugin-statusbar 2.0.0-dev "StatusBar"
cordova-plugin-whitelist 1.0.0 "Whitelist"
cordova-plugin-x-socialsharing 5.0.8-dev "SocialSharing"
org.apache.cordova.network-information 0.2.15 "Network Information"
